firewood/src/merkle/node.rs

```diff
@@ -526,39 +286,45 @@ impl Node {
 
 impl Storable for Node {
     fn deserialize<T: CachedStore>(addr: usize, mem: &T) -> Result<Self, ShaleError> {
-        const META_SIZE: usize = 32 + 1 + 1;
+        const META_SIZE: usize = TRIE_HASH_LEN + 1 + 1;
+
         let meta_raw =
             mem.get_view(addr, META_SIZE as u64)
                 .ok_or(ShaleError::InvalidCacheView {
                     offset: addr,
                     size: META_SIZE as u64,
                 })?;
-        let attrs = meta_raw.as_deref()[32];
+
+        let attrs = meta_raw.as_deref()[TRIE_HASH_LEN];
```

Comment:
every use of `meta_raw `calls `as_deref()`. Can we move this to the assignment?

Reply:
All this code is going away in a future PR